<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtHelpFiltersDialog</class>
 <widget class="QDialog" name="QtHelpFiltersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage QtHelp Filters</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filters:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Attributes:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QListWidget" name="filtersList">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QTreeWidget" name="attributesList">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>1</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="addButton">
       <property name="toolTip">
        <string>Press to add a new filter</string>
       </property>
       <property name="text">
        <string>Add Filter ...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="removeButton">
       <property name="toolTip">
        <string>Press to remove the selected filters</string>
       </property>
       <property name="text">
        <string>Remove Filters</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="removeAttributeButton">
       <property name="toolTip">
        <string>Press to remove the selected attributes</string>
       </property>
       <property name="text">
        <string>Remove Attributes</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="unusedAttributesButton">
       <property name="statusTip">
        <string>Press to select all unused attributes</string>
       </property>
       <property name="text">
        <string>Select Unused</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>filtersList</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>attributesList</tabstop>
  <tabstop>removeAttributeButton</tabstop>
  <tabstop>unusedAttributesButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QtHelpFiltersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
